### PR TITLE
Language, grammar and text clarity updates.

### PR DESCRIPTION
**Pull request by mikeymakesit**
I updated templates with grammar, spelling and some very minor clarity updates.  I hope you find this useful.

### CODE REVIEW

**Comment by NotExpectedYet** [comment]
Hey thanks for this! 

I will take a look through as soon as I can. 

I've had a quick browse and looks great so far. 

Would you mind rebasing to the betas/beta-1.8.0 branch please?

**Comment by mikeymakesit** [comment]
Sure I'll rebase to "betas/beta-1.8.0" and update the PR.

**Comment by mikeymakesit** [comment]
I rebased to betas/beta-1.8.0 and updated the PR to merge into that branch.

File: server/templates/partials/PrinterManager/healthChecks.modal.ejs [hunk 1]
```diff
@@ -12,40 +12,37 @@
         <div class="col-lg-12">
             <div class="collapse" id="collapseExample">
                 <div class="alert alert-secondary" role="alert" id="healthChecksHelpText">
-                    <p>OctoFarm will periodically run a few checks on your printers just to make sure nothing is out of
-                        whack! Each check will either pass (Green icon) or fail (Red icon). Clicking a red icon will
-                        give you an information box on the problem, OctoFarms reaction and finally what solution you can
+                    <p>OctoFarm will periodically run checks on your printers to make sure nothing is out of
+                        whack! Each check will either pass (reen icon) or fail (ed icon). Clicking a red icon will
```
Comment (NotExpectedYet) [inline server/templates/partials/PrinterManager/healthChecks.modal.ejs]: Should be Green and Red

**Review by NotExpectedYet — CHANGES_REQUESTED** [review]
Good work here man, had to have a laugh at some of that stuff I've left in the app for ages god... Eslapsed was my favourite! 

Just one change, I found G and R missing from a colour. 

Big thank you for taking your time to do that.

File: server/templates/partials/PrinterManager/healthChecks.modal.ejs (hunk 1 above)
Comment (mikeymakesit) [reply server/templates/partials/PrinterManager/healthChecks.modal.ejs]: Ah great catch.  I've added a commit to address that.

**Pull request merged**